.github/workflows/ci.yml

```diff
@@ -68,3 +68,34 @@ jobs:
         cp -r build/ ./bwc-test/
         cd bwc-test/
         ./gradlew bwcTestSuite -Dtests.security.manager=false
+
+  build-artifact-names:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - id: security-plugin-version
+      uses: madhead/read-java-properties@66cc8c88f5c6f6069ebfb42586024dd6ffe2e451
```

Comment:
will this long string change at any point in future leading to a breaking change?